docker-compose.yml

```diff
@@ -4,7 +4,6 @@ services:
 
   postgres:
     image: circleci/postgres:11.6-alpine-ram
-    restart: always
```

Comment:
Removed these to default to "no" as it would restart the pods whenever I start my laptop regardless of whether I wanted them running.